modules/nf-core/pretextsnapshot/main.nf

```diff
@@ -0,0 +1,31 @@
+process PRETEXTSNAPSHOT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::pretextsnapshot=0.0.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pretextsnapshot:0.0.4--h7d875b9_0':
+        'quay.io/biocontainers/pretextsnapshot:0.0.4--h7d875b9_0' }"
+
+    input:
+    tuple val(meta), path(pretext)
```

Comment:
```suggestion
    tuple val(meta), path(pretext)
```

Maybe called `pretext` a `map` instead? or `pretext_map`? as this corresponds with the option